composer.json

```
@@ -86,7 +86,7 @@
         "league/flysystem-cached-adapter": "^1.0"
     },
     "require-dev": {
-        "allure-framework/allure-phpunit": "~1.2.0",
+        "allure-framework/allure-phpunit": "~1.3.0",
```

Version 1.3.0 doesn't contain the fix for the bug you're trying to fix here.
```suggestion
        "allure-framework/allure-phpunit": "~1.3.1",
```